packages/devextreme-scss/scss/widgets/fluent/list/_sizes.scss

```diff
@@ -3,6 +3,8 @@
 // adduse
 
 $fluent-list-vertical-padding: 4px !default;
+$fluent-list-horizontal-padding: 4px !default;
+$fluent-list-item-checkbox-horizontal-margin: 8px !default;
```

Comment:
I assume this value is incorrect in compact mode. It will 4px.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28809/commits/da51a9aa161a1a7b434d529f4a7b1a2b3ee1b201